include/aikido/planner/Planner.hpp

```diff
@@ -44,6 +49,9 @@ class Planner
 protected:
   /// State space associated with this planner.
   statespace::ConstStateSpacePtr mStateSpace;
+
+  /// RNG the planner uses.
+  std::unique_ptr<common::RNG> mRng;
```

Comment:
If we're taking in a raw `RNG` pointer, do we need to store this as a `unique_ptr`?